### ISSUE TITLE
Zoom & Rotate [Version 3 enhancement]

### ISSUE THREAD

**Issue by khesed** [noah-nuebling/mac-mouse-fix#664]
## Description
The Click and Drag action current only has two options: "Spaces & Mission Control" and "Scroll & Navigate"
Please consider adding a third "Zoom & Rotate" option, where vertical dragging does zooming in and out, and horizontal dragging does rotation (right clockwise, left counter-clockwise).

## Rationale
There are no mouse gestures which can do rotation.

**Comment by noah-nuebling** [comment]
Hey there @khesed, thanks for your feedback. You can zoom and rotate through a Click and Scroll action. Do you have a usecase where it would be better to have a Click and Drag action?

**Comment by khesed** [comment]
> You can zoom and rotate through a Click and Scroll action. 

Hmm, I only see Zoom; I don't see any rotate option:
<img width="229" alt="image" src="https://github.com/noah-nuebling/mac-mouse-fix/assets/74698406/6279a14f-067c-4f21-93d5-76ce5f8c0c8d">

> Do you have a usecase where it would be better to have a Click and Drag action?

I'm not sure exactly how rotating would be intuitively handled with scrolling. Since Rotate would only need one axis of Click and Drag, I think it pairs nicely with Zoom. The gestures are already very similar (and usually co-activated) on trackpad.

It would be intuitive to me if mouse movements up could zoom in, down could zoom out, right could clockwise rotate, left could counter-clockwise rotate.

**Comment by philippedcote** [comment]
> Hey there @khesed, thanks for your feedback. You can zoom and rotate through a Click and Scroll action. Do you have a usecase where it would be better to have a Click and Drag action?

I also agree with the proposed idea. I came from a Logitech mouse which had this exact feature. Now I'm using a SteelSeries which doesn't have this and I really, really miss it.

A simple Click & Drag for Zoom In or Out would also be very nice.

Thanks.
